metadata-models/src/main/pegasus/com/linkedin/dataset/DatasetExpectations.pdl

```diff
@@ -0,0 +1,14 @@
+namespace com.linkedin.dataset
+
+/**
+ * Expectations about a dataset
+ */
+@Aspect = {
+  "name": "datasetExpectations"
+}
+record DatasetExpectations {
```

Comment:
Expectations is a very Great Expectations-Specific terminology. Can we call this aspect DatasetAssertions? Or DatasetValidations?